sys-boot/uefi-mkconfig/uefi-mkconfig-1.4.ebuild

```
@@ -0,0 +1,35 @@
+# Copyright 2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit linux-info optfeature
+
+DESCRIPTION="Automatic management of UEFI entries"
+HOMEPAGE="https://github.com/Biosias/uefi-mkconfig"
+SRC_URI="https://github.com/Biosias/uefi-mkconfig/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64"
+
+RDEPEND="
+	app-shells/bash
+	sys-boot/efibootmgr
+"
+
+CONFIG_CHECK="EFI_STUB"
+
+src_install() {
+	dobin uefi-mkconfig
+	einstalldocs
+}
+
+pkg_postinst() {
+	elog "uefi-mkconfig: Automatic management of UEFI entries"
+	elog "Run uefi-mkconfig while having all efi partitions mounted"
+	elog "Please use with care, this package was tested on a limited number of machines"
+	elog "Some problems may arise due to different implementations of UEFI"
+	elog ""
+	optfeature "Regenerating UEFI configuration on each kernel installation" \ "sys-kernel/installkernel[-systemd,efistub]"
```

```suggestion
	elog 
	optfeature "Regenerating UEFI configuration on each kernel installation" \ 
	     "sys-kernel/installkernel[-systemd,efistub]"
```

We don't need to give an empty string argument to elog to print an empty line. And the optfeature line we can make a bit shorter, so pkgcheck does not complain.